src/ol/DataTile.js

```diff
@@ -75,21 +75,26 @@ class DataTile extends Tile {
    * @api
    */
   load() {
-    this.state = TileState.LOADING;
-    this.changed();
+    if (this.state == TileState.ERROR) {
+      this.state = TileState.IDLE;
+    }
+    if (this.state == TileState.IDLE) {
+      this.state = TileState.LOADING;
+      this.changed();
```

Comment:
This seems a bit mysterious.

If all you want to do is prevent loading if not idle, I would add these lines to the start of the function:
```js
if (this.state !== TileState.IDLE) {
    return;
}
```

Reply:
Never mind, I see the behavior is similar for image tiles.